mongo-helm/templates/mongo-express.yml

```diff
@@ -0,0 +1,51 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mongo-express
+  labels:
+    app: {{ .Values.mongoexpress.appName }}
+spec:
+  replicas: 1

```

Comment:
We should be able to set replica count from values file. Please don't hard code such things.

Reply:
Fixed.